src/pages/graphql/schema/orders/mutations/cancel-order.md

```diff
@@ -0,0 +1,54 @@
+---
+title: cancelOrder mutation | Commerce Web APIs
+---
+
```

Comment:
Add the following lines above the head1 and be sure to include a blank line before the heading.

```
import BetaNote from '/src/_includes/graphql/notes/beta.md'

<BetaNote />
```